drools-benchmarks-parent/README.md

```diff
@@ -0,0 +1,13 @@
+Drools benchmarks
+==========================
+
+The benchmarks are written using [JMH](http://openjdk.java.net/projects/code-tools/jmh/).
+
+This module contains multiple Maven modules. The instructions how to run individual benchmark modules are described in the README files of the modules:
+- drools-benchmarks
+  - Variety of benchmarks, benchmarks are grouped by package names. May have longer run time.  
+- drools-benchmarks-quick
+  - Quick benchmarks that are supposed to be used for initial sanity measurements. 
```

Comment:
We will probably need to be a bit more precise with `quick` definition :)
We can also see if `OptaPlanner` team is interested in a similar approach

Reply:
We can align the description as the quick benchmarks will be added in future PRs. Would that be ok?

Reply:
Fine for me 👍